### PR TITLE
MOBILE Modeled Coverage updates

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

**Comment by samgutentag** [comment]
KaTeX equations will require PR #1066 to be merged to render correctly

File: docs/5g-on-helium/mobile-poc.mdx [hunk 1]
```diff
@@ -36,302 +49,360 @@ factors:
...
+green res8 hex and a purple res12 hex inside.
+
+<img
+  className="explorerSidebar"
+  src={useBaseUrl('img/5g-on-helium/hex-12-and-hex-8-comparison.png')}
+/>
+
+### Modeled Coverage Reward Tiers
+
+:::info Heartbeat impact on Radio rewards
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Heartbeat impact title seemed to be unrelated to the explainer text.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 2]
```diff
@@ -371,13 +445,158 @@ Latency     150 ->  `Fail`
...
+:::info
+
+It may take up to 12 hours for newly connected 5G Hotspots to perform a Speed Test, and a minimum of
+2 Speed Tests are required for the MOBILE PoC Oracle to calculate a moving average.
+
+:::
+
+<Tabs block={true}>
+
+<TabItem value="successful speed test" label="Successful Speed Test" default>
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Would it make sense to add status icons like you have for other toggle buttons for different outcomes?

**Review by meowshka — DISMISSED** [review]

**Comment by samgutentag** [comment]
> 

this is complete

**Comment by samgutentag** [comment]
ignore that close/reopen 😆

File: docs/5g-on-helium/mobile-poc.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/5g-on-helium/mobile-poc.mdx]: agreed, removed!

File: docs/5g-on-helium/mobile-poc.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/5g-on-helium/mobile-poc.mdx]: yes, good call, adding icons and reordering the tabs to move from "good" -> "not good" as well

**Review by meowshka — APPROVED** [review]

**Pull request merged**